hack/build-static-site.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# build-static-site.sh
+# Generates a static HTML site into _site/
+# Requires Go ...and Ruby
```

Comment:
I think it would be useful to make this available to run with a docker container (e.g. with a `make serve` command that does a `docker build / docker run`) for developers to test locally.